iracema/aggregation.py

```diff
@@ -5,8 +5,8 @@
 import numpy as np
 
 from iracema.util.windowing import apply_sliding_window
-from iracema.timeseries import TimeSeries
-
+#from iracema.timeseries import TimeSeries
```

Comment:
I recommend you to leave this module as it was before. It is not necessary to import the whole `timeseries` module if only the class `TimeSeries` is being used. And avoid commenting out lines to deactivate the code.